src/Ninja/G1CP/Content/Fixes/Gamesave/fix234_DE_LogEntryDrax.d

```diff
@@ -0,0 +1,60 @@
+/*
+ * #234 Spelling - Drax: "Orkhund" (DE)
+ */
+func int G1CP_234_DE_LogEntryDrax() {
+    const string curString = "Wissen �ber Z�hne entfernen - Wolf, Orchund, Snapper, Beisser, Bluthund, Schattenl�ufer.";
+    const string newString = "Wissen �ber Z�hne entfernen - Wolf, Orkhund, Snapper, Bei�er, Bluthund, Schattenl�ufer.";
+    const string topicName = "";
+
+    // Do only once per session for performance on consecutive calls
+    const int topicId = -2; // -1 is reserved for invalid symbols
+    const int count   = -1;
+    if (topicId == -2) {
+        // Find and retrieve the topic
+        topicId = G1CP_GetStringConstId("GE_AnimalTrophies", 0);
+        topicName = G1CP_GetStringConstI(topicId, 0, "");
+
+        // Replace the push of the old string with the new string (this is never reverted, i.e. session fix)
+        count = G1CP_ReplacePushStr(
+            G1CP_GetFuncId("Org_819_Drax_Creatures_Zahn", "void|none"), 0, curString, newString
+        );
```

Comment:
This breaks the coding style of function calls a bit. I would suggest either one of these instead:
```D
        count = G1CP_ReplacePushStr(G1CP_GetFuncId("Org_819_Drax_Creatures_Zahn", "void|none"), 0,
                                    curString, newString);
```
or better yet, more readable and more consistent with the existing code
```D
        var int symbId; symbId = G1CP_GetFuncId("Org_819_Drax_Creatures_Zahn", "void|none");
        count = G1CP_ReplacePushStr(symbId, 0, curString, newString);
```